js/binance.js

```diff
@@ -6043,6 +6044,56 @@ module.exports = class binance extends Exchange {
         };
     }
 
+    async repayMarginLoan (code, symbol, amount, params = {}) {
+        await this.loadMarkets ();
+        let market = undefined;
+        if (symbol !== undefined) {
+            market = this.market (symbol);
+        }
+        const currency = this.currency (code);
+        const request = {
+            'asset': currency['id'],
+            'amount': amount,
+        };
+        const isIsolated = this.safeValue (params, 'isIsolated');
```

Comment:
We should unify this as `defaultMarginType` with support for an option and a param-override, see how that is done in okx and huobi.